web/packages/teleport/src/services/resources/types.ts

```diff
@@ -56,9 +57,10 @@ export type Role = {
     deny: RoleConditions;
     options: RoleOptions;
   };
-  version: string;
 };
 
+export type RoleVersion = 'v3' | 'v4' | 'v5' | 'v6' | 'v7';
```

Comment:
Can we define role versions once and reuse them? These are also defined in the `standardmodal.ts` file

Reply:
The only way I see to do it is to convert it to an enum, as you suggested. I can do it, I just thought that in the end, it would be more code for little purpose, especially that in general, we tend to err on the side of using union types instead of enums for these things.

Reply:
This is fine without the enum imo. it wont allow invalid options with the way its typed anyway